HPXMLtoOpenStudio/tests/test_constructions.rb

```diff
@@ -57,10 +54,11 @@ def test_windows_shading
         assert_equal(sf_winter, winter_transmittance)
       end
 
-      # Check view factor for ground diffuse
-      default_view_factor = 0.5 # Default for vertical wall
-      view_factor = default_view_factor * (sf_summer * summer_length + sf_winter * winter_length) / 12.0
-      assert_in_delta(view_factor, os_window.viewFactortoGround.get, 0.001)
+      # Check subsurface view factor to ground
+      subsurface_view_factor = 0.5
+      os_window = model.getEnergyManagementSystemActuators.select { |w| w.actuatedComponent.get.name.to_s == window.id }[0]
```

Comment:
Name should be updated, this is no longer a window (subsurface) object.

Reply:
Oops. Yep I will change the name.